rc/core/c-family.kak

```diff
@@ -154,7 +154,7 @@ evaluate-commands %sh{
 }
 
 # c specific
-add-highlighter shared/c/code/numbers regex %{\b-?(0x[0-9a-fA-F]+|\d+)[fdiu]?|'((\\.)?|[^'\\])'} 0:value
+add-highlighter shared/c/code/numbers regex %{\b-?(0x[0-9a-fA-F]+|\d+)([uU][lL]{1,2}|[uU]|[lL]{1,2}|[lL]{1,2}[uU]|[fFdDiI])?|'((\\.)?|[^'\\])'} 0:value
```

Comment:
I think this can be compacted into `[uU][lL]{0,2}|[lL]{1,2}[uU]?|[fFdDiI]`, is that right ?

Reply:
Yes. Seem's like I've gone too far at some point. I've tested your proposed regex, it works fine. I'll commit it to my branch. Thanks.